Misc/Pair.java

```diff
@@ -0,0 +1,50 @@
+class IntIntPair implements Comparable<IntIntPair> {
```

Comment:
Wasn't this file already merged [here](https://github.com/vanshikaarora/DS_Algo/pull/19/files#diff-fb953f72ba2705f9a4d1920911cd5128)

Reply:
Sorry, my bad.